modules/exploits/multi/php/ignition_laravel_debug_rce.rb

```diff
@@ -0,0 +1,222 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = AverageRanking
```

Comment:
Could you elaborate on the ranking? `AverageRanking` is reserved for "unreliable or difficult to exploit" issues (https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking). I don't necessarily see that reflected in any documentation.

Reply:
My bad. Classical copy and paste from other module. I'll change it to ExcellentRanking.